### PR TITLE
Ddf 3690 remove redundant config

### PR DESCRIPTION
**Pull request by sblackmore**
#### Who is reviewing it? 
@mcalcote 
@vinamartin 
#### Select relevant component teams: 
https://github.com/orgs/codice/teams
#### Choose 2 committers to review/merge the PR. 
(please choose ONLY two committers from below, delete the rest)
@coyotesqrl
@ricklarsen - Documentation
@rzwiefel 

#### How should this be tested? (List steps with links to updated documentation)

- Log in to DDF as admin
- Go to Admin Console -> Admin -> Admin UI Configuration
- Verify that the Header and Footer fields are no longer present
- Go to Admin Console -> Platform -> Configuration -> Platform UI Configuration
- Verify that the Header and Footer fields are still present
- Change the Header and Footer values and Save
- Refresh the page and make sure that the new Header and Footer appear

#### Checklist:
- [x] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

**Comment by coyotesqrl** [comment]
Confirming that you have [installed the githooks](https://codice.atlassian.net/wiki/spaces/DDF/pages/70986756/Cloning+Building+DDF) for the DDF repository. The commit messages [do not look correctly formatted](https://codice.atlassian.net/wiki/spaces/DDF/pages/6291518/Git+Commit+Message+Guidelines).

**Comment by vinamartin** [comment]
❓ I think this change looks fine, but I'm a little unclear on what the nuance is between both of these configurations.
See similar issue: https://github.com/codice/ddf/pull/2724, where I looked up `ddf.platform.ui.config` to find the configuration for Simple Search UI instead of it having its own configuration. 
What is the point of having these two configurations and what is the difference between them? Is this something that should be addressed with this change or future improvements to configuration in our system?

**Comment by sblackmore** [comment]
To me it would make sense to have a single place to set most of these
configurations, but the difference between them is that the configuration
in the Admin UI app only pertains to the Admin UI. The Header and Footer
fields configured under the Platform app change the Header and Footer for
both Intrigue and the Admin Console, but the Header and Footer fields
configured under the Admin app don't change Headers and Footers for either
Intrigue or the Admin console (hence removing them). I could potentially
see why you might want to have separate System Usage Messages, so for now
having separate configuration for each makes sense

**Comment by mcalcote** [comment]
@Lambeaux would you be willing to comment on this convo 👆

**Comment by vinamartin** [comment]
@sblackmore That makes sense to me. ✅

**Comment by mcalcote** [comment]
build now

File: platform/admin/ui/src/main/resources/OSGI-INF/metatype/metatype.xml [hunk 1]
```diff
@@ -39,17 +39,6 @@
...
-        <AD
-                description="Specifies the header text to be rendered on the Administrator Console."
-                name="Header" id="header" required="true" type="String"
-                default=""
-                />
-        <AD
-                description="Specifies the footer text to be rendered on the Administrator Console."
-                name="Footer" id="footer" required="true" type="String"
-                default=""
-                />
```
Comment (Lambeaux) [inline platform/admin/ui/src/main/resources/OSGI-INF/metatype/metatype.xml]: ✅ These are under `platform/admin` and I believe are the correct configs to remove. They weren't serving any value. A hero of this PR should verify banners can still be configured, **and** that any custom guest claims profiles are still executed by the installer correctly.

**Comment by Lambeaux** [comment]
Build now.

**Comment by vinamartin** [comment]
❗️ I manually tested this change. I'm not seeing that Style and Text Color are being used either. Can you verify this @sblackmore ?

**Comment by sblackmore** [comment]
@vinamartin I was wondering also if I should go ahead and take those fields out too as part of this ticket. I can do it no problem

**Comment by vinamartin** [comment]
Manually verified this change ✅ LGTM!

**Comment by vinamartin** [comment]
build now

**Comment by sblackmore** [comment]
build now

**Comment by vinamartin** [comment]
build now

**Comment by Lambeaux** [comment]
Build now.

**Pull request merged**